config/deploy.rb

```diff
@@ -4,7 +4,7 @@
 #################
 set :application, 'dashboard'
 # set branch based on env var or ask with the default set to the current local branch
-set :branch, ENV['branch'] || ENV['BRANCH'] || ask(:branch, `git branch`.match(/\* (\S+)\s/m)[1])
+set :branch, Figaro.env.BRANCH || ask(:branch, `git branch`.match(/\* (\S+)\s/m)[1])
```

Comment:
These are for being able to override stuff, ex `BRANCH=mycustombranch cap staging deploy` to do a branch deploy, so I'd leave them